src/alr/alr-commands-publish.adb

```diff
@@ -48,10 +63,23 @@ package body Alr.Commands.Publish is
    is
       use GNAT.Command_Line;
    begin
+      Define_Switch
+        (Config,
+         Cmd.Commit'Access,
+         "", "--commit=",
+         "Revision in a remote repository denoting the sources of a release",
+         Argument => "ID");
+
       Define_Switch (Config,
                      Cmd.Hash'Access,
                      "", "--hash",
```

Comment:
I am not sure we still need the `--hash` feature here since it only does a sha512.

Reply:
Yes, we can deprecate this.